src/web/modules/transactions/refund.njk

```diff
@@ -2,7 +2,7 @@
 
 {% extends "layout/layout.njk" %}
 
-{% set isTestData = not transaction.live %}
+{% set isTestData = not transaction.parent and transaction.parent.live %}
```

Comment:
you're pretty confident in your operator precedence here. Some brackets wouldn't go amiss?

Reply:
With nunjucks operator precedence on top of JavaScripts questionable operator precedence brackets would go a long way, thanks for the suggestion!